README.md

```diff
@@ -1,16 +1,23 @@
 # custom-event-polyfill
 
-Polyfill for creating CustomEvents on IE9/10/11 if native implementation is missing.
+Polyfill for creating CustomEvents on IE9/10/11 if native implementation is missing (or misbehaves).
 
 ## Install
 
-```
+```sh
 $ npm install custom-event-polyfill
 ```
 
-## Useage 
+## Usage 
+
+```js
+// ES5
+require('custom-event-polyfill')
+// ES2015
+import CustomEvent from 'custom-event-polyfill'
```

Comment:
IMO, I wouldn't include the es6 style import.  It implies that the module exports a definition, which it doesn't because it's a polyfill, not ponyfill.